addons/project/models/project_task.py

```diff
@@ -807,7 +807,7 @@ def default_get(self, default_fields):
             project = self.env['project.project'].browse(project_id)
             if project.analytic_account_id:
                 vals['analytic_account_id'] = project.analytic_account_id.id
-        elif 'default_user_ids' not in self.env.context:
+        elif 'default_user_ids' not in self.env.context and 'user_ids' in default_fields:
             user_ids = vals.get('user_ids', [])
             user_ids.append(Command.link(self.env.user.id))
             vals['user_ids'] = user_ids
```

Comment:
Nitpicking, feel free to ignore
```suggestion
            vals['user_ids'].append(Command.link(self.env.user.id))
```

Reply:
Why? there is no guarantee that the returned dict of `defaut_get` holds a value for each field.
`vals['user_id']` could crash :thinking:

Reply:
really? damn I'm a noob x)